backend/post-service/src/main/java/com/speakapp/postservice/services/ReactionService.java

```diff
@@ -37,4 +41,26 @@ public ReactionType createUpdatePostReaction(ReactionType newReaction, UUID post
         return newReaction;
     }
 
+    public ReactionType createUpdateCommentReaction(ReactionType newReaction, UUID commentId, UUID userId){
+        Comment comment = commentRepository.findById(commentId).orElseThrow(() -> new ResponseStatusException(HttpStatus.NOT_FOUND,
+                "Comment with provided commentId" + commentId + "does not exist"));
```

Comment:
wiem że się czepiam szczegółów ale tutaj ci się sklei i wyjdzie coś "Comment with provided commentId036fd207-b7a7-48d3-bf43-a45f8faa31cfdoes not exist"